react/tic-tac-toe/src/app/components/Topbar/index.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/button-has-type */
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import actionsCreators from '../../../redux/Login/actions';
+import userLogo from '../../assets/user-logo.svg';
+
+import styles from './styles.module.scss';
+
+class Topbar extends Component {
+  logout = () => {
+    this.props.logout();
```

Comment:
You can remove this `logout` method and just pass the prop to the `onClick`.
Then The component should be changed from a class to a function.